caasp-environment.yaml

```diff
@@ -13,3 +13,14 @@ parameters:
   # hostnames won't be resolved.
   internal_net_cidr: 172.28.0.0/24
   dns_nameserver: 172.28.0.2
+  # cpi parameters
+  cpi_auth_url: https://192.168.24.26:5000/
+  cpi_domain_name: Default
+  cpi_tenant_name: admin
+  cpi_region: region1
+  cpi_username: admin
+  cpi_password:
+  cpi_lb_mon_retries: 3
+  cpi_bs_version: 1
```

Comment:
In `caasp-stack.yaml` tempate it is `cpi_cinder_bs_version`

Reply:
Valid values are v1, v2, v3 and auto

Reply:
thx! done added constraint, fixed parameter name